pytd/client.py

```diff
@@ -77,6 +77,7 @@ def __init__(
         self.database = database
 
         self.default_engine = default_engine
+        self.query_executed = None
```

Comment:
It'd be better to have docstring under `Attributes` section with adding `Examples` to get job_id.

Reply:
Added the description: 3b17a704269f7c6c1c47c4e99c9fa14217c46f69